src/schemas/json/partial-taskipy.json

```diff
@@ -0,0 +1,123 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://json.schemastore.org/partial-taskipy.json",
+  "title": "taskipy",
+  "description": "tasks runner for python projects",
+  "type": "object",
+  "properties": {
+    "tasks": {
+      "title": "tasks",
+      "description": "task definitions",
+      "type": "object",
+      "patternProperties": {
+        "^[0-9a-zA-Z]+$": {
```

Comment:
this is too strict.

it marks example tasks from the official repo as invalid eg: https://github.com/taskipy/taskipy?tab=readme-ov-file#composing-tasks

it also disallows things like:
```
[tool.taskipy.tasks]
"lint:fix" = "ruff check --fix"
```

Reply:
https://github.com/SchemaStore/schemastore/pull/4517

Reply:
thank you

Reply:
I should probably add this case to [CONTRIBUTING/Avoiding Overconstraint](https://github.com/SchemaStore/schemastore/blob/master/CONTRIBUTING.md#avoiding-overconstraint).